uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/views/CheckoutView.kt

```diff
@@ -466,6 +481,58 @@ internal class CheckoutView @JvmOverloads constructor(context: Context,
         }
     }
 
+    private fun bindComments(comments: String?){
+        bookingCheckoutCommentView.setActionIcon(R.drawable.kh_uisdk_ic_checkout_comments)
+
+        bookingCheckoutCommentView.setTitle(context.getString(R.string.kh_uisdk_checkout_comments_title))
+        comments?.let {
+            bookingComments = it
+            if(it.isEmpty())
+                bookingCheckoutCommentView.setSubtitle(context.getString(R.string.kh_uisdk_checkout_comments_subtitle))
+            else
+                bookingCheckoutCommentView.setSubtitle(it)
+        }?: kotlin.run {
+            bookingCheckoutCommentView.setSubtitle(context.getString(R.string.kh_uisdk_checkout_comments_subtitle))
+        }
+    }
+
+    private fun showCommentsDialog(){
+        val commentBottomSheet = CheckoutCommentBottomSheet().apply {
+            onCommentsChanged = {
+                bindComments(it)
+            }
+            bookingComments.isNotEmpty().let {
+                initialComments = bookingComments
+            }
+        }
+        commentsListener?.invoke(commentBottomSheet)
+    }
+
+    private fun bindTravelDetails(){
+        val capabilities = presenter.getCurrentQuote()?.fleet?.capabilities
+        if(isPrebook){
+            if(capabilities?.any { x -> x == CapabilityAdapter.FLIGHT_TRACKING } == true && (presenter.getJourneyDetails()?.pickup?.details?.type == PoiType.AIRPORT)){
```

Comment:
Instead of any, use contains since it doesn't require comparison with boolean

Reply:
Switched to contains, but still need the == true because of nullable check